getusers.md

```diff
@@ -16,10 +16,13 @@ We also need a result for each channel. A count from each Group will be fine.
 ## examples, which result i think what we need
 
 ```JSON
-	{
-		"UserID" : "integer",
-    		"ChannelID" : "integer",
-    		"ConnectionID" : "integer",
-	}
+{
+  "from": "String",
+  "to": ["String"],
+  "msg": "String",
+  "method": "String",
+  "createdAt": "Datetime",
+  "userlist" : ["String"],
```

Comment:
Do we really need another field in the payload?

Reply:
I think yes, because the from fields is for my opinion for messages, which will be sent to an another user. Or can we use the same field "to" for a list of users? Maybe that the user which will send a message can have a drop down list to search for the user? If so, we don't need a another field